cmd/juju/controller/addmodel.go

```diff
@@ -55,20 +55,30 @@ type addModelCommand struct {
 
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
-To add a model, you must at a minimum specify a model name. You may
-also supply model-specific configuration, a credential, and which
-cloud/region to deploy the model to. The cloud/region and credentials
-are the ones used to create any future resources within the model.
 
+To add a model, you must specify a model name. 
 Model names can be duplicated across controllers but must be unique for
 any given controller. Model names may only contain lowercase letters,
 digits and hyphens, and may not start with a hyphen.
 
-Credential names are specified either in the form "credential-name", or
-"credential-owner/credential-name". There is currently no way to acquire
-access to another user's credentials, so the only valid value for
-credential-owner is your own user name. This may change in a future
-release.
+To add a model, Juju requires a credential.
+If a controller has a credential for the user creating the model, 
+it will be used.
+If a controller does not have a credential, 
+this command allows to specify desired named credential with --credential option.
+This option stores provided named credential in the controller.
+
+Another way of ensuring that a named credential is stored in the controller, is
```

Comment:
I don't think this is particularly relevant at the moment. We'll probably want to change the backend code so that if there's just one credential for a user, we'll use that; but that is not the case at the moment. Even if it were, I think it would be more useful to explain a scenario in which that would be helpful.

Please drop this sentence, unless you have an argument against the above.